### ISSUE TITLE
AttributeList parser misses all properties

### ISSUE THREAD

**Issue by diegoquinteiro** [ts-graphviz/ts-graphviz#756]
## Describe the bug

When parsing graphs or subgraphs with node, edge or graph attribute lists, all resulting attribute lists are empty

## To Reproduce

```typescript
const model = fromDot('digraph { edge [ style="invis" ]; a -> b }');
console.log(model.attributes.edge); // AttributeList will be empty;
```

## Expected behavior

```typescript
console.log(model.attributes.edge); // AttributeList should contain style = invis
```

## Additional context

It seems like the issue is here:

https://github.com/ts-graphviz/ts-graphviz/blob/c1e4c50a9eaca7f5a511eca3d520ae9a02271844/src/ast/model-shim/to-model/plugins/utils/apply-statments.ts#L45-L55

As `stmt.kind` will be in lower-case and not match the proper switch case.

**Comment by kamiazya** [comment]
@all-contributors please add @diegoquinteiro for bug.

**Comment by kamiazya** [comment]
I fixed this issue and release patch as [v1.3.4](https://github.com/ts-graphviz/ts-graphviz/releases/tag/v1.3.4).

Thanks!

**Comment by diegoquinteiro** [comment]
Works like a charm! Thank you so much =)
